examples/guidelines.md

```diff
@@ -0,0 +1,78 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# Example Guidelines
+
+## An Example Is
+
+An example demonstrates running a real application/framework/workload
+on Kubernetes in a meaningful way. It is educational and informative.
+
+Examples are not:
+
+* Full app deployments, ready to use, with no explanation. These
+  belong either
+  [here](https://github.com/kubernetes/deployment-manager) or in
+  something like [Helm](https://helm.sh/).
+* Simple toys to show how to use a Kubernetes feature. These belong in
+  the [user guide](../docs/user-guide/).
+* Demos that follow a script to show a Kubernetes feature in
+  action. Example: killing a node to demonstrate controller
+  self-healing.
+
+## An Example Includes
+
+### Up front
+
+* Should have a "this is what you'll learn" section.
```

Comment:
And classify the type of workload: web app, storage, big data, batch/workflow, etc. Example categorization:
http://mesos.apache.org/documentation/latest/frameworks/

And classify by whether it's Introductory, Intermediate, Advanced. We may want a special category for examples that assume the user is new to Docker/containers, as well (Newbie?).